packages/devtools_app/lib/src/flutter/connect_screen.dart

```diff
@@ -93,9 +95,8 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
                 // of hard coding material colors.
                 borderSide: BorderSide(width: 0.5, color: Colors.grey),
               ),
-              hintText: 'http://127.0.0.1:12345/AUTH_CODE=/',
+              hintText: 'http://127.0.0.1:12345/auth_code=',
```

Comment:
Fyi @jwren  is this consistent with how you are displaying the hint text for auth codes in Intellij?

Reply:
I made changes in ~Q4 to both this hint text and the hint text in the Dart Plugin for IntelliJ to be consistent (including the capitalization).  If we think it looks better with lower case auth_code, I'll change it on the Dart Plugin side.

Reply:
I think it does look better. Likely useful for this to be in sync with other tools (if pretty low-priority for things like case consistency).